<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMonitorControl</class>
 <widget class="QDialog" name="DialogMonitorControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="cbStartHumidity">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Humidity</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbStartTemperature">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Temperature</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbStartLuminosity">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string> Start Luminosity</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogMonitorControl</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMonitorControl</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
